README.md

```diff
@@ -17,54 +17,171 @@ Helm is a package manager for Kubernetes that allows you to easily install and m
-  organisation: MYORGANISATION # Set this to the name of your organisation. It will be used across the deployment for branding and naming.
-  secure: false # Set this to true if you want to serve ACS over HTTPS. This is recommended for production deployments but can be turned off for development.
-  tlsSecretName: factoryplus-tls # Set this to the name of the secret containing the wildcard certificate for the above domain. This is only required if secure is set to true.
-identity:
-  realm: FACTORYPLUS.MYORGANISATION.COM # Set the identity realm for the deployment. This is used to namespace the identity server and should be unique to your deployment. It is recommended that you use the baseUrl in capitals for this value.
+Finally, install ACS by running the following command.
+```bash
+helm install acs amrc-connectivity-stack/amrc-connectivity-stack --version $version -f $values -n $namespace --create-namespace
+```
+
+Replace the `$variables` appropriately:
+
+|Variable|Value|
+|---|---|
+|`$version`     | The version of ACS to install. |
+|`$values`      | The path to your `values.yaml` file created above. |
+|`$namespace`   | The Kubernetes namespace to use. |
+
+The version number should be a specific version like `1.0.2` or a semver
+requirement string like `^2.0.0`.
+
+If all went to plan you should now have a fully functioning ACS deployment beginning to deploy to your local Kubernetes cluster. Note that it can take a few minutes to have all services operational.
+
+Take note of the service URLs printed at the end of the installation. You will need these to connect to the various services.
+
+### Upgrade to a released version of ACS
+
+If you have an existing ACS install and you want to upgrade to a new
+release, first make sure you check the release notes for the release.
+If you are upgrading over a major version increment there may be manual
+steps required. We do not support upgrades that skip major versions; if
+you want to upgrade from N.x.y to N+2.x.y you need to upgrade to the
+latest N+1.x.y first.
+
+You can follow the same procedure to re-deploy with different settings
+in your `values.yaml` file.
+
+Run
+
+```bash
+helm repo update
 ```
 
-Before we install, we need to create the `factory-plus` namespace, which is where all ACS services will be deployed to. If a different namespace is chosen by changing the `-n <namespace>` on the helm install command then ensure the namespace exists before installing ACS.
+to pull down any updates to the Helm chart. Then run
 
-To create the `factory-plus` namespace, run the following command:
 ```bash
-kubectl create namespace factory-plus
+helm upgrade acs amrc-connectivity-stack/amrc-connectivity-stack -f $values --version $version
 ```
 
-Finally, install ACS by running the following command.
+specifying the version to upgrade to. You need to use the same `$values`
+file as before, but you don't need to re-specify the namespace.
```

Comment:
Is this the case? I'm sure the namespace is required but I could be wrong.

Reply:
You are definitely wrong :). I use `helm upgrade -f values.yaml acs ..\amrc-connectivity-stack` regularly.